multiversx_sdk_core/utils.py

```diff
@@ -0,0 +1,11 @@
+from pathlib import Path
+
+from multiversx_sdk_core.errors import ErrBadFile
+
+
+def read_binary_file(path: Path) -> bytes:
+    try:
+        with open(path, 'rb') as binary_file:
+            return binary_file.read()
+    except Exception as err:
+        raise ErrBadFile(str(path), err)
```

Comment:
Perhaps it's fine (sufficient) to allow the original error to be thrown, without being wrapped in the extra `ErrBadFile`?

Reply:
Now returning only the original error. Deleted the `ErrBadFile`